src/bundler/bundle_v2.zig

```diff
@@ -9520,13 +9535,26 @@ const LinkerContext = struct {
             );
 
             switch (c.options.source_maps) {
-                .external => {
+                inline .external, .linked => |tag| {
```

Comment:
Does this need the `inline`?
```suggestion
                    .external, .linked => |tag| {
```